src/aws_client_sup.erl

```diff
@@ -0,0 +1,42 @@
+%%%-------------------------------------------------------------------
+%% @doc aws_client top level supervisor.
+%% @end
+%%%-------------------------------------------------------------------
+
+-module('aws_client_sup').
```

Comment:
Yeah, we have to add them in templates because people try names like `abc-def`which are invalid atoms without quoting. For style it should be taken out. I might try to improve the templating to figure it out on its own though  and then remove default quotes.